java-checks/src/main/java/org/sonar/java/checks/GarbageCollectorCalledCheck.java

```diff
@@ -19,18 +19,22 @@
  */
```

Comment:
Nice simplification! This is a perfect example to show that some rules are implemented since already quite some years, with the changes and improvement of the plugin, it is now possible to refactor some rule to something way cleaner.